src/include/partitioning/custom_part_dialogs.rb

```diff
@@ -1674,8 +1674,7 @@ def SubvolHandling(old, new)
             )
             Popup.Message(tmp)
             pth = svtmp + pth
-          end
-          if Builtins.contains(SubvolNames(new), pth)
+          elsif Builtins.contains(SubvolNames(new), pth)
```

Comment:
Sure, I will take a look at it. Thank you for pointing out the issue, I missed it when I was testing my change :)